lego/apps/achievements/models.py

```diff
@@ -25,3 +25,14 @@ def percentage(self):
             .count()
         )
         return (achievement_users / total_users) * 100
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["user", "identifier"], name="unique_user_identifier"
+            )
+        ]
+        indexes = [
+            models.Index(fields=["identifier", "user"]),
```

Comment:
```suggestion
```
Is this redundant? If I've understood this correctly, I believe the unique constraint (`UniqueConstraint(fields=["user", "identifier"])`) automatically creates a unique index on these fields in postgres, and the second index `models.Index(fields=["identifier", "user", "level"])` can already handle queries that only need `identifier` and `user`

Reply:
Ah yeah that makes sense! You're right